pkg/services/serviceaccounts/extsvcaccounts/service_test.go

```diff
@@ -380,3 +380,64 @@ func TestExtSvcAccountsService_SaveExternalService(t *testing.T) {
 		})
 	}
 }
+
+func TestExtSvcAccountsService_RemoveExtSvcAccount(t *testing.T) {
+	extSvcSlug := "grafana-test-app"
+	tmpOrgID := int64(1)
+	extSvcAccID := int64(10)
+	tests := []struct {
+		name   string
+		init   func(env *TestEnv)
+		slug   string
+		checks func(t *testing.T, env *TestEnv)
+		want   *extsvcauth.ExternalService
+	}{
+		{
+			name: "should do nothing",
```

Comment:
Add a little more verbosity like: should not add a service account and fails to retrieve a service account that doesn't exists